physx/src/articulation_link.rs

```diff
@@ -235,15 +235,15 @@ impl ArticulationLinkBuilder {
         &self,
         body: &mut ArticulationReducedCoordinate,
         parent: PartHandle,
-        joint_transform: Option<glm::Mat4>,
+        joint_transform: Option<Mat4>,
     ) -> PartHandle {
-        let parent_quat = nalgebra::UnitQuaternion::from_euler_angles(
-            self.parent_rotation.x,
-            self.parent_rotation.y,
-            self.parent_rotation.z,
+        let parent_quat = Quat::from_rotation_ypr(
```

Comment:
Could you verify and do a test to see if the rotations are equivalent between nalgebra and glam here? glam is left-handed and may apply the rotations in a different order so something to verify

Reply:
Yeah I'll add a few test cases. There are quite a few transformations that I have to do, I need to make sure that I am doing the correct thing.

Reply:
Previously this used `1 roll − 2 pitch − 3 yaw`, but the function here use yaw pitch roll.

But this is never used anyway, so I'll replace this `Vec3` which was implicitly roll pitch yaw with a `Quat`.